### PR TITLE
Simplify running of movie reviews model

### PR DESCRIPTION
**Pull request by loostrum**
by making the tokenizer part of the model

### CODE REVIEW

File: example_data/model_generation/movie_reviews/generate_model.ipynb [hunk 1]
```diff
@@ -307,13 +295,12 @@
...
     "        input_data, label = batch\n",
-    "        input_data = input_data.to(device)\n",
-    "        label = label.to(device)\n",
+    "        label = torch.FloatTensor(label).to(device)\n",
     "        \n",
     "        optimizer.zero_grad()\n",
     "        predictions = model(input_data).squeeze(1)\n",
     "        \n",
-    "        loss = loss_func(predictions, label)\n",
+    "        loss = loss_func(predictions, label.long())\n",
```
Comment (egpbos) [inline example_data/model_generation/movie_reviews/generate_model.ipynb:303]: Is the `.long()` necessary?

**Review by egpbos — COMMENTED** [review]
Looks good! Indeed simpler.

File: example_data/model_generation/movie_reviews/generate_model.ipynb (hunk 1 above)
Comment (loostrum) [reply example_data/model_generation/movie_reviews/generate_model.ipynb:303]: It is, apparently this loss function doesn't like shorter floats.

Comment (egpbos) [reply example_data/model_generation/movie_reviews/generate_model.ipynb:303]: Ok then! :D

**Comment by loostrum** [comment]
Turns out that extra functions added to the model class are lost when converting to onnx and we have been using the model as-is for a long time
